doc/advanced-testing-mini-course/src/Lec05/History.hs

```diff
@@ -15,7 +15,14 @@ import Lec05.StateMachine
 
 ------------------------------------------------------------------------
 
-newtype History = History (TQueue HistEvent)
+newtype History = History (TQueue HistEvent')
+
+data Dropped = DidDrop | DidArrive
+
+data HistEvent' = HistEvent'
```

Comment:
Is this wrapper type helping in any way compared to merely adding `Dropped` to `HistEvent`?

Reply:
Just saw the commit message... hmm